src/mock_crust/support.rs

```diff
@@ -278,7 +283,10 @@ impl ServiceImpl {
     }
 
     pub fn whitelist_peer(&mut self, peer_id: PeerId) {
-        let _ = self.whitelist.insert(peer_id);
+        if !self.whitelist.insert(peer_id.clone()) {
+            debug!("Duplicate insert attempt  whitelist for peer : {:?}",
```

Comment:
Duplicate whitespace insert attempted! :grin: 
(Between `attempt` and `whitelist`.)